Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="denys.kotovskiy@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` should be set to `/app` instead of `app/` to ensure that the path is absolute. This is a common convention and helps avoid potential issues with relative paths.